### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by IknowJoseph**
A few typo fixes and readability enhancements for the readme file.

### CODE REVIEW

File: cadasta/templates/core/base.html [hunk 1]
```diff
@@ -153,27 +153,27 @@
           <div class="footer-links">
             <ul class="list-inline">
               <li>
-                <a href="http://cadasta.org/about-us/">
+                <a href="http://cadasta.org/about-us/" target="_blank">
```
Comment (oliverroick) [inline cadasta/templates/core/base.html]: Not sure about `target="_blank"` here. I personally don't like when new tabs or windows open unexpectedly. What do you think, @clash99 ?

Comment (IknowJoseph) [inline cadasta/templates/core/base.html]: Great, thanks,

To be honest, the base.html changes were made to my fork of the platform, then accidentally included in this pull request... It was only the readme changes I was going for, and won't be upset if the base.html changes get ignored.

Comment (clash99) [inline cadasta/templates/core/base.html]: I agree @oliverroick - let's not add the target change yet.  I'll add it to a future consideration list.

**Comment by oliverroick** [comment]
@IknowJoseph I tried to rebase your branch onto our latest master, but I made a mistake and I'm not able to proceed with your PR. I'm very sorry about that.

Your changes to README are not lost; I have copied them into a [gist](https://gist.github.com/oliverroick/2ff299d920013f39079080f94d27a573), feel free to open a new PR. Make sure that you create a new branch from master in your cloned repository, because that will make rebasing your branch much more easy for us. Thanks and apologies again.

**Pull request closed without merging**